Source/SwiftLintBuiltInRules/Rules/Lint/FunctionArgumentsSpacingRule.swift

```diff
@@ -0,0 +1,38 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FunctionArgumentsSpacingRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "functions_arguments_spacing",
+        name: "Function Arguments Spacing",
+        description: "Remove the space before the first function argument and after the last argument",
+        kind: .lint
+    )
+}
+
+private extension FunctionArgumentsSpacingRule {
+    final class Visitor: ViolationsSyntaxVisitor<ConfigurationType> {
+        override func visitPost(_ node: FunctionCallExprSyntax) {
+            let argsCount = node.arguments.count
+            guard argsCount != 0 else {
+                return
+            }
+
+            let leftParanTrailingTrivia = node.leftParen?.trailingTrivia
```

Comment:
It's sometimes a bit surprising to which node SwiftSyntax attaches the whitespace. Therefore, you should also check the leading trivia of the first argument and the trailing trivia of the last argument.